### PR TITLE
Boot order handling

### LINKED ISSUE #85: Fix boot order handling
The way we handle boot order is deprecated and should be fixed.

### The problem
Proxstar is still using the old boot order format, which means that when it creates a new vm, proxmox sees no boot hard disk, and it will reboot into the installer ad nauseam. This is fixable, you can manually set the boot order in the vm's options page, but in doing so, proxstar won't be able to read the boot order, and thus the vm won't be viewable at all from proxstar, so users will need a proxmox account with at least PVEVMConsole permissions to access the vm, and PVEVMUSER to edit anything.

### Stuff you probably need to fix this
d44698a959d02986d4400f80fbd13c5fdbe6b9cd will need to be reverted, and then the boot order handling form, route, and methods in `vm.py` will need to be adapted.

See [the qm conf file docs](https://pve.proxmox.com/wiki/Manual:_qm.conf) for the `boot` parameter to see what needs to be handled.

### PR DESCRIPTION
**Pull request by harmonherring**
Fixes #85 
May need adjustments for mobile devices, possibly remove device description in the selection modal

### CODE REVIEW

**Review by galenguyer — APPROVED** [review]

**Comment by WillNilges** [comment]
I'd like to get this working on my local machine before we merge this, if I can actually find a single second in my day to actually do so.

File: proxstar/static/js/script.js
Comment (WillNilges) [inline proxstar/static/js/script.js:932]: According to @harmonherring the boot order selection menu descriptions break the view on mobile devices.

**Review by WillNilges — APPROVED** [review]
I have verified that the changes work. @harmonherring has a known issue with the mobile view that he'll fix. As soon as that's pushed, I'll merge this PR.

**Comment by WillNilges** [comment]
So, not having this is causing a lot of issues in Proxstar right now.
```
2021-11-11 19:20:47,395 ERROR Exception on /vm/217 [GET]
Traceback (most recent call last):
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/app-root/lib64/python3.8/site-packages/flask_pyoidc/flask_pyoidc.py", line 253, in wrapper
    return view_func(*args, **kwargs)
  File "/opt/app-root/src/proxstar/__init__.py", line 220, in vm_details
    return render_template(
  File "/opt/app-root/lib64/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/opt/app-root/lib64/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/opt/app-root/src/proxstar/templates/vm_details.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/opt/app-root/src/proxstar/templates/base.html", line 90, in top-level template code
    {% block body %}
  File "/opt/app-root/src/proxstar/templates/vm_details.html", line 21, in block "body"
    {% for interface in vm.interfaces %}
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
  File "/opt/app-root/src/proxstar/util.py", line 18, in _lazy_property
    setattr(self, attr_name, fn(self))
  File "/opt/app-root/src/proxstar/vm.py", line 153, in interfaces
    ip = get_ip_for_mac(starrs, mac)
  File "/opt/app-root/src/proxstar/starrs.py", line 20, in get_ip_for_mac
    c.execute('BEGIN')
psycopg2.OperationalError: SSL connection has been closed unexpectedly
```

Gonna merge this to make it stop dying. @galenguyer and I aren't 100% sure if it'll help, but ¯\_(ツ)_/¯

**Pull request merged**
